tap_linkedin_ads/streams.py

```diff
@@ -282,7 +282,7 @@ def adanalyticscolumns(self) -> list[str]:
             "viralLandingPageClicks,viralExternalWebsitePostClickConversions,externalWebsiteConversions,viralVideoFirstQuartileCompletions,leadGenerationMailContactInfoShares,clicks,viralClicks,shares,viralFullScreenPlays,videoMidpointCompletions,viralCardClicks,viralExternalWebsitePostViewConversions,viralTotalEngagements,viralCompanyPageClicks,actionClicks,viralShares,videoCompletions,comments,externalWebsitePostViewConversions,dateRange",
             "costInUsd,landingPageClicks,oneClickLeadFormOpens,talentLeads,sends,viralOneClickLeadFormOpens,conversionValueInLocalCurrency,viralFollows,otherEngagements,viralVideoCompletions,cardImpressions,leadGenerationMailInterestedClicks,opens,totalEngagements,videoViews,viralImpressions,viralVideoViews,commentLikes,pivot,viralLikes",
             "adUnitClicks,videoThirdQuartileCompletions,cardClicks,likes,viralComments,viralVideoMidpointCompletions,viralVideoThirdQuartileCompletions,oneClickLeads,fullScreenPlays,viralCardImpressions,follows,videoStarts,videoFirstQuartileCompletions,textUrlClicks,pivotValue,reactions,viralReactions,externalWebsitePostClickConversions,viralOtherEngagements,costInLocalCurrency",
-            "viralVideoStarts,viralRegistrations,viralJobApplyClicks,viralJobApplications,jobApplications,jobApplyClicks,viralExternalWebsiteConversions,postViewRegistrations,companyPageClicks,documentCompletions,documentFirstQuartileCompletions,documentMidpointCompletions,documentThirdQuartileCompletions,downloadClicks,viralDocumentCompletions,viralDocumentFirstQuartileCompletions,viralDocumentMidpointCompletions,viralDocumentThirdQuartileCompletions,viralDownloadClicks,impressions",
```

Comment:
@lucas5thorneRyanMiranda are these meaning to remove the `viralDocumentThirdQuartileCompletions` column? I didnt see anything noted in the PR description and it looks like that column is still referenced in the schema https://github.com/MeltanoLabs/tap-linkedin-ads/pull/63/files#diff-d3caa667c1dc7e3ada59d3189681817406a6dba0c45df80d33407bcb0ed05dc3L251.

Reply:
Hey Pat,

Good catch. This column was replaced by approximateUniqueImpressions due to the limit of fields we can pull from, but was only meant as a stand-in to test that we are pulling approximateUniqueImpressions. We added this column back and actually removed the pivot column, which is not a part of the fivetran table and is a static value (“Campaign” or “Creative”) every time.

Reply:
@NeilGorman104 that makes sense. Thanks for sharing that context, very helpful!